### ISSUE TITLE
How to get the pre-trained model？

### ISSUE THREAD

**Issue by Fino2020** [awsm-research/VQM#1]
While executing the `run_train.sh` the parameter `--pretrained_model_name=vqr_pretrained_model.bin --model_name=vqr_fine_tuned_model.bin`, which means the pretrained model name.  However, I can not find the model or any description of the model in your repository. Could you please tell me how to download this model please

**Comment by mmmmmyl** [comment]
> While executing the `run_train.sh` the parameter `--pretrained_model_name=vqr_pretrained_model.bin --model_name=vqr_fine_tuned_model.bin`, which means the pretrained model name. However, I can not find the model or any description of the model in your repository. Could you please tell me how to download this model please

Do you succeed finally, I can't reproduce the experiment results if I train from codebert and codet5 without the pretrained model

**Comment by Fino2020** [comment]
After executing download_model.sh, it seems to download a pre-trained model, but with a different name than the one given, you can try it. I ran it about 6 months ago but I don't remember now, I can go back and try it then share it if needed.

**Comment by mmmmmyl** [comment]
> After executing download_model.sh, it seems to download a pre-trained model, but with a different name than the one given, you can try it. I ran it about 6 months ago but I don't remember now, I can go back and try it then share it if needed.

I used the 'loc_fine_tuned_model', and train the 'vqr_model' by myself, but I can't get the correct rate of 33.21% in the paper ,I tried to change parameters but I can not get any result better than 29%. Can you provide some detailed parameters like weight_decay, random seed. Appreciate for any help you can provide!

**Comment by Fino2020** [comment]
I was also unable to achieve the results mentioned in the paper, I achieved even worse results, barely less than 20%, I'm not quite sure what the reason for this is, but in my paper I used my reproduced results with some explanations.

**Comment by HMF2024** [comment]
> I was also unable to achieve the results mentioned in the paper, I achieved even worse results, barely less than 20%, I'm not quite sure what the reason for this is, but in my paper I used my reproduced results with some explanations.

Hi, I'm having a similar problem right now. What their paper says is that they pre-trained on the bug-fix dataset from the 'Neural Transfer Learning for Repairing Security Vulnerabilities in C Code' paper, and the 'Vision Transformer- Inspired Automated Vulnerability Repair' paper provides the same bug-fix dataset as in 'Neural Transfer Learning for Repairing Security Vulnerabilities in C Code' paper. ', do you know why, or have you encountered a similar problem?

**Comment by HMF2024** [comment]
> I was also unable to achieve the results mentioned in the paper, I achieved even worse results, barely less than 20%, I'm not quite sure what the reason for this is, but in my paper I used my reproduced results with some explanations.

Hi, I am experiencing a similar issue. Their paper states that they pre-trained on the Bug-fix dataset from the paper “Neural Transfer Learning for Repairing Security Vulnerabilities in C Code.” However, the Bug-fix dataset provided in the open-source code of “Vision Transformer-Inspired Automated Vulnerability Repair” is different from the one I downloaded from the open-source code of “Neural Transfer Learning for Repairing Security Vulnerabilities in C Code.”

In their Bug-fix dataset, CWE types are even specified, whereas in the original Bug-fix dataset, all CWE types are uniformly labeled as CWE-000. Do you know why this is the case, or have you encountered a similar issue?
